system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_unstuck.rs

```diff
@@ -47,6 +49,9 @@ impl OnRuntimeUpgrade for UnstuckSnowbridge {
 		if !is_bridge_stuck() {
 			return RocksDbWeight::get().reads(1);
 		}
+		if is_bridge_upgraded() {
```

Comment:
This is unnecessary and adds confusion. The upgrade should only run if the bridge is stuck according to `is_bridge_stuck`.